rpmalloc/rpmalloc.c

```diff
@@ -681,6 +688,9 @@ static atomic32_t _huge_pages_current;
 static int32_t _huge_pages_peak;
 #endif
 
+static const char _memory_huge_page_name[] = "rpmalloc Huge Page";
+static const char _memory_page_name[] = "rpmalloc Page";
```

Comment:
I would prefer if these were set through the config. Instead of an int flag, perhaps const char* that if set is used to name the pages?